src/navigation/menu.es6

```diff
@@ -50,8 +50,11 @@ export const extractChartableMarkets = (trading_times_data) => {
 };
 
 export const refreshMenu = (root, markets, callback) => {
+   const non_crypto = symbol => !/^(cry|JD)/i.test(symbol);
```

Comment:
Please use a more descriptive name for this function!

Reply:
sure